molo/core/models.py

```diff
@@ -41,6 +41,12 @@ class SectionPage(Page):
         index.SearchField('description'),
     )
 
+    @property
```

Comment:
Silly, I was looking at the wagtail demo app on how to do stuff. I'll remove this decorator